tests/proxy/clone_invalid_filter.t

```diff
@@ -53,6 +53,11 @@
 
   $ cd ${TESTTMP}
   $ git clone -q http://localhost:8002/real_repo.git:sub1.git
-  remote: JoshError(invalid filter)
+  remote: Invalid filter command: sub1
```

Comment:
I would phrase is like this:
`Invalid filter: ":sub1"`
It's not really a command, and the `:` should` be included also to be more consistent with the suggested fix (which includes the `:` )